Sources/UIExtensions/ViewModifiers/View+MeasureView.swift

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2022 Lautsprecher Teufel GmbH. All rights reserved.
+
+import SwiftUI
+
+struct ViewSizeKey: PreferenceKey {
+    static let defaultValue: CGSize = .zero
+
+    static func reduce(value: inout CGSize, nextValue: () -> CGSize) {
+        value = nextValue()
```

Comment:
nitpicking: reduce is meant to solve conflicts when you have 2 siblings that both publish ViewSizeKey changes. In this case, the last one will always will, so VStack { something().measureView(); somethingElse().measureView() } will produce weird stuff.

Reply:
That works well. From the call's to the closure you can see that SwiftUI is measuring the views from outside to the inside. See https://github.com/teufelaudio/UIExtensions/blob/2c17eb9453d5ddccae2f0263db73ca0763b08a7a/Tests/UIExtensionsTests/__Snapshots__/View%2BMeasureViewTests/testMeasureView-SE-light.1.png and https://github.com/teufelaudio/UIExtensions/blob/2c17eb9453d5ddccae2f0263db73ca0763b08a7a/Tests/UIExtensionsTests/View%2BMeasureViewTests.swift#L11

Reply:
True... Maybe not for siblings then, but for nested.
```swift
VStack {
    HStack {
       Text("").measureView { a in }
    }.measureView { b in }
}.measureView { c in }
```
Question, how reliable is "c"?